### ISSUE TITLE
Bad indentation reporting false positive when using tabs

### ISSUE THREAD

**Issue by Australis86** [PyCQA/pylint#5893]
### Bug description

I use tabs for my indentation and whilst setting the indent-string to "t" causes pylint to correctly identify the number of indents, it still reports bad identation, e.g.:

```python
#! /usr/bin/python

import os
try:
	import Levenshtein
except ImportError as err:
	Levenshtein = None
	print("Levenshtein module not installed. Spellchecking will not be available.")

print("Hello Word")
```

### Command used

```shell
pylint -f colorized --function-naming-style='camelCase' --indent-string="t" a.py  --enable=all
```

### Pylint output

```shell
************* Module test
test.py:4:0: W0311: Bad indentation. Found 1 spaces, expected 0 (bad-indentation)
test.py:5:0: W0311: Bad indentation. Found 1 spaces, expected 1 (bad-indentation)
test.py:7:0: W0311: Bad indentation. Found 1 spaces, expected 1 (bad-indentation)
test.py:8:0: W0311: Bad indentation. Found 1 spaces, expected 1 (bad-indentation)
test.py:11:0: C0304: Final newline missing (missing-final-newline)
test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
test.py:3:0: W0611: Unused import os (unused-import)
test.py:5:1: W0611: Unused import Levenshtein (unused-import)

--------------------------------------------------------------------
Your code has been rated at -1.43/10 (previous run: -1.43/10, +0.00)
```

### Expected behavior

```shell
************* Module test
test.py:11:0: C0304: Final newline missing (missing-final-newline)
test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
test.py:3:0: W0611: Unused import os (unused-import)
test.py:5:1: W0611: Unused import Levenshtein (unused-import)
```

### Pylint version

```shell
$ pylint --version
pylint 2.12.2
astroid 2.9.3
Python 3.9.10 (main, Jan 20 2022, 21:37:52)
[GCC 11.2.0]
```

**Comment by Pierre-Sassoulas** [comment]
I can reproduce, but the issue is with the command you launch, you need to use ``--indent-string="\t"`` not ``--indent-string="t"``:

```diff
- pylint -f colorized --function-naming-style='camelCase' --indent-string="t" a.py  --enable=all
+ pylint -f colorized --function-naming-style='camelCase' --indent-string="\t" a.py  --enable=all
```

**Comment by Australis86** [comment]
Thank you; I can't believe I missed that!
